### DISCUSSION TITLE
Could bias toward operators at end of line in long conditions be relieved?

### DISCUSSION THREAD

**Discussion by gwhitney** [DanielXMoore/Civet#690] [Ideas]
Right now, if I need to break a long assignment, I can do it either before or after an operator:

```
total = summand1 +
   summand2

product = factor1
   * factor2;
```
compiles to
```
total = summand1 + summand2;
product = factor1 * factor2;
```
That's nice, because it lets me use the second arrangement, and I am personally a believer in the arguments out there that the "operators just after linebreaks" style is clearer and less error prone.

However,
```
if test1 and
      test2
   result
```
compiles fine to `if (test1 && test2) { result }` whereas
```
if test1
      and test2
   result
```
is a parse error. Since it is a parse error, could Civet be modified to allow the latter form as well, and compile it into the same code (so that I can have my beloved operators-after-linebreaks pattern back ;-) ?

Thanks for considering.

**Comment by STRd6** [comment]
Yeah, we should probably allow that. Thanks for opening the discussion!

**Reply by edemaine** [reply]
Implemented in #1483!
